packages/app/src/app/pages/common/Modals/PreferencesModal/Experiments/BetaSandboxEditor.tsx

```diff
@@ -100,6 +102,7 @@ export const BetaSandboxEditor = () => {
               slowness: false,
               navigability: false,
               other: false,
+              feedback: null,
```

Comment:
if you are passing an extra value to rows, you need to change the param of the POST request, the range is now A:G instead of A:F on line 25